src/containers/Login.vue

```diff
@@ -115,58 +99,23 @@
       </div>
 
       <!-- SEMESTERS -->
-      <div class="container-fluid p-0 small">
-        <div class="row semester phonepad m-0">
-          <div class="col-12 col-md-5 comment">
+      <div class="container-fluid p-0 center">
+        <div class="semester phonepad row no-gutters float-right">
+          <div class="comment comment-sem">
             <h1 class="head">Plan Your Semesters</h1>
-            <p class="sub">
+            <p class="sub-text">
               Use CoursePlan’s semesterly planner to choose courses well in advance and ensure that
               you never miss a requirement
             </p>
           </div>
-          <div class="col-md-7 image-wrapper image-wrapper--semester">
-            <img class="hide schedule" src="@/assets/images/schedule.svg" alt="Plan preview" />
-          </div>
-        </div>
-      </div>
-
-      <!-- FIRST -->
-      <div class="container justify-content-center phonepad first m-0">
-        <div class="container inside">
-          <div class="row justify-content-center">
-            <div class="col justify-content-center">
-              <h1 class="head-center text-center">Be The First One To Use It</h1>
-              <p class="sub text-center" style="padding: 30px">
-                Gain early access by filling out your email below and help us grow into what you
-                need!
-              </p>
-            </div>
-          </div>
-          <div class="row justify-content-center">
-            <div class="col-12 col-md-12 email m-0 pb-0">
-              <input
-                class="waitlist-info"
-                type="text"
-                placeholder="Your Email Address"
-                v-model="waitlist.email"
-              />
-            </div>
-            <div class="col-12 col-md-12 email m-0">
-              <input
-                class="waitlist-info"
-                type="text"
-                placeholder="Your Major"
-                v-model="waitlist.major"
-              />
-            </div>
-            <div class="col-12 col-md-12 email">
-              <button class="email-button" variant="primary" @click="addUser()">
-                Join Waitlist
-              </button>
-            </div>
-          </div>
+          <img
+            class="hide schedule"
+            src="@/assets/images/landing_new/img2.svg"
+            alt="Plan preview"
+          />
         </div>
       </div>
+      <br style="clear: both" />
```

Comment:
What does this do?

Reply:
It clears the float he used to make "Plan Your Semesters" go to the right